NEWS.md

```diff
@@ -117,6 +117,8 @@
 
 14. `by=...get()...` could fail with `object not found`, [#4873](https://github.com/Rdatatable/data.table/issues/4873) [#4981](https://github.com/Rdatatable/data.table/issues/4981). Thanks to @sindribaldur for reporting, and @OfekShilon for fixing.
 
+15. Using a plain non-ASCII symbol as `by` works as expected, [#4708](https://github.com/Rdatatable/data.table/issues/4708). Thanks @pfv07 for the report.
```

Comment:
I've been trying to place code/example/function call as the first word in each news item, and rather than saying 'as expected' stating what was wrong before or what was expected. In my mind I try to imagine users with code bases that use `data.table` reading the news items and wondering how they're going to find out if they need to check if they unknowingly have been impacted by the bug.
 So in this case, how about :
> `dt[,,by=año]` (using a column name containing a non-ASCII character) no longer errors with  `object 'año' not found`, [#4708](https://github.com/Rdatatable/data.table/issues/4708). Thanks @pfv07 for the report, and Michael Chirico for the fix.

By telling users it was an error before, we assuage their fears that they've been getting a silent wrong result before.
By putting code first in news items, it's faster to grok.  The reader can _see_ the non-ASCII character in the middle of a column name, which conveys much more, more quickly.

I know I can just make this change myself as I have been doing. But now I've changed to request-change instead, as a way to convey knowledge and thinking like this.

Reply:
Makes sense, thanks!